content/participants/ttjc@tarunyadav1/index.md

```diff
@@ -34,8 +34,18 @@ I need a mentor to create connections in the tech industry.
 
 ### projects
 
-Coming soon...
+#### **FIRST PROJECT**
+
+    A Movie Library App made with ReactJS using The Movie Database API.
```

Comment:
Hey! @tarunyadav1 Please remove space between the left margin and your text in your project and blog description. Using tabs in markdown converts the normal text into code blocks.

Reply:
Thank you @khusharth. I will remove Tabs. Do I need to create another pull request?

Reply:
No. You can just add another commit in this branch.

Reply:
@khusharth  I had remove Tab. Kindly accept my request. Once again thank you.